<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareScanner</class>
 <widget class="QMainWindow" name="ShareScanner">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ShareScanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SSres.qrc">
    <normaloff>:/icons/icons/intranet.png</normaloff>:/icons/icons/intranet.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP Range: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_start">
        <property name="inputMask">
         <string>000.000.000.000; </string>
        </property>
        <property name="text">
         <string>172.16.12.2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip_end">
        <property name="inputMask">
         <string>000.000.000.000; </string>
        </property>
        <property name="text">
         <string>172.16.15.255</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="SSres.qrc">
          <normaloff>:/icons/icons/zmedia-playback-start.png</normaloff>:/icons/icons/zmedia-playback-start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Space, Return</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopbutton">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="SSres.qrc">
          <normaloff>:/icons/icons/zmedia-playback-pause.png</normaloff>:/icons/icons/zmedia-playback-pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>File Manager</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="chooseFileManager">
        <item>
         <property name="text">
          <string>nautilus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dolphin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>konqueror</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="chooseSpeed">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>fast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>slow</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="indentation">
       <number>20</number>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>IP Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hostname</string>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MAC Address</string>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Response Time</string>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>Scan</string>
    </property>
    <addaction name="actionNewScan"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="actionCheckUpdate"/>
   </widget>
   <addaction name="menuScan"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCollapseAll"/>
   <addaction name="actionExpandAll"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExpandAll">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/tree_diagramm_new.png</normaloff>:/icons/icons/tree_diagramm_new.png</iconset>
   </property>
   <property name="text">
    <string>expand all</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/gnome_edit_clear.png</normaloff>:/icons/icons/gnome_edit_clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="toolTip">
    <string>clear</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/tree_diagramm_delete.png</normaloff>:/icons/icons/tree_diagramm_delete.png</iconset>
   </property>
   <property name="text">
    <string>collapse all</string>
   </property>
  </action>
  <action name="actionNewScan">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/gnome_edit_clear.png</normaloff>:/icons/icons/gnome_edit_clear.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/edit-delete.png</normaloff>:/icons/icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/user-identity.png</normaloff>:/icons/icons/user-identity.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCheckUpdate">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/intranet.png</normaloff>:/icons/icons/intranet.png</iconset>
   </property>
   <property name="text">
    <string>Check for an update</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/gtk-find-and-replace.png</normaloff>:/icons/icons/gtk-find-and-replace.png</iconset>
   </property>
   <property name="text">
    <string>find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/zmedia-playback-start.png</normaloff>:/icons/icons/zmedia-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="SSres.qrc">
     <normaloff>:/icons/icons/zmedia-playback-pause.png</normaloff>:/icons/icons/zmedia-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SSres.qrc"/>
 </resources>
 <connections/>
</ui>
